code/renderers/vue3/package.json

```diff
@@ -62,7 +63,7 @@
     "@types/prettier": "2.7.2",
     "@vitejs/plugin-vue": "^4.4.0",
     "typescript": "^5.3.2",
-    "vue": "^3.2.47",
+    "vue": "^3.4.5",
```

Comment:
this change has not impact as vue is peerDeps and the latest patch will be installed anyway